website/docs/components/code-editor/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,108 @@
+## Usage
+
+### When to use
+
+- When a user needs to create code from scratch.
+- When code requires modification from an existing source.
+
+### When not to use
+
+- As a read-only reference for code, use the [Code Block](/components/code-block) instead.
+
+## Standalone
+
+The `isStandalone` property increases the portability of the Code Editor to ensure that it can be used in different contexts. For example, a common use case of the Code Editor is in a "standalone" context, which can be part of a form, multi-step process, and is generally a part of the normal layout flow.
```

Comment:
```suggestion
The `isStandalone` property increases the portability of the Code Editor to ensure that it can be used in different contexts. For example, a common use case of the Code Editor is in a "standalone" context, which can be part of a form or multi-step process and is generally a part of the normal layout flow.
```